CONTRIBUTING.md

```diff
@@ -48,6 +48,7 @@ This contribution policy will evolve over time. For now it is based on a slightl
 6. A PR **SHOULD** avoid "callback-hell" style and instead prefer "async/await" style.
 7. A PR **MUST** follow the requirements spelled out in this project's [Style Guide](docs/Style-Guide.md).
 8. A PR **MUST** receive approval from at least one reviewer (existing, long-term contributor) before being merged. *Reviewers may not review their own PRs, and may not push commits to someone else's PR.*
+8. A PR **MUST** receive approval from the designer when it's related to the user interface before being merged.
```

Comment:
@mmbotelho since you are helping so much in the review process, I thought it would be nice to reflect it properly in our docs. What do you think?

Reply:
I would love it, thank you for doing that! My work only ends when you guys implement it, so I think it is fundamental that I (or any other designers that might work in the project) get to review UI implementations.

Reply:
I love this idea as well! Nice job. Just make sure it's marked as rule `9.` (not `8.` which already exists)

Reply:
Please note, you may want to merge review (and wait for merging of) PR #670, which updates rule 8

Reply:
Fixed!